operators/transformation/mergemap.md

```diff
@@ -12,6 +12,20 @@
 
 ---
 
+### Why use `mergeMap`?
+
+`mergeMap` is for you when you want to observe all action from the source's emission
```

Comment:
This is too technical I think, when would you use it? For http requests? What type? Why would you use this over switchMap?